### ISSUE TITLE
Use Clarity for SPAs (Vuejs)

### ISSUE THREAD

**Issue by theogilardo** [microsoft/clarity#95]
Hello, I am trying to install Clarity on my app, a SPA built with Nuxt.js. Since it is a SPA, all the resources required to navigate through the website are loaded on the first page load. The application will often update the URL in the address bar to emulate traditional page navigation, but another full page request is never made. Therefore, the snippet code is run only once for the first page which is not very convenient. 

Is there a way to update the Microsoft Clarity tracker manually? That way, each time a route changes, I would update manually the tracker and get data on the page the user is actually browsing in.

**Comment by alessandro-riz** [comment]
I've the same problem on a emberjs app

**Comment by elliothogg** [comment]
Any update on this issue?

**Comment by RenPaw** [comment]
Any update on this issue?

**Comment by nargespms** [comment]
Any update on this issue?

**Comment by squpshift** [comment]
Hello?

**Comment by ender336** [comment]
Clarity should be marking pages in SPAs when the url changes (even if another server request isn't made). If you have specific issues with that not working can you clarify?

**Comment by AbdelrhmanMagdy** [comment]
Since we haven't received any further updates or responses, we're closing this issue for now. If you’re still encountering 
the problem, feel free to reopen it or open a new issue with more details.
